lib/archethic/oracle_chain/scheduler.ex

```diff
@@ -284,13 +308,16 @@ defmodule ArchEthic.OracleChain.Scheduler do
         prev_pub,
         next_pub
       )
-      |> ArchEthic.send_new_transaction()
 
-      Logger.debug("New data pushed to the oracle")
-    end
+    Task.start(fn -> ArchEthic.send_new_transaction(tx) end)
+
+    Logger.debug("New data pushed to the oracle",
+      transaction_address: Base.encode16(tx.address),
+      transaction_type: :oracle
+    )
   end
 
```

Comment:
maybe we can add spec & some doc here and summary

Reply:
works fine

Reply:
> maybe we can add spec & some doc here and summary

Forgot the put in private . Thanks